bigbluebutton-html5/public/locales/en.json

```diff
@@ -440,6 +441,17 @@
     "app.poll.liveResult.secretLabel": "This is an anonymous poll. Individual responses are not shown.",
     "app.poll.removePollOpt": "Removed Poll option {option}",
     "app.poll.emptyPollOpt": "Blank",
+    "app.poll.quiz": "Quiz",
+    "app.poll.quiz.error": "Quiz has not correct answers selected. Please select at least one correct answer.",
+    "app.poll.quiz.start.label": "Start Quiz",
+    "app.poll.quiz.showCorrectAnswer": "Show correct answer to all users",
+    "app.poll.quiz.options.title": "Quiz options",
+    "app.poll.quiz.options.tooltip": "Select this answer as the correct one",
+    "app.poll.quiz.options.correct": "Correct Answer",
+    "app.poll.quiz.options.incorrect": "Incorrect Answer",
+    "app.poll.quiz.liveResult.title.correct": "Correct",
+    "app.poll.quiz.options.responseNotSecret": "Quiz – the presenter can see your answer.",
```

Comment:
Is the final period in this translation intentional? as it doesn’t seem to be the standard to include one